src/SFA.DAS.IdentifyDataLocks.Domain/CollectionPeriodReport.cs

```diff
@@ -39,8 +39,8 @@ public class CollectionPeriodReport
             IEnumerable<EarningEventModel> earnings)
         {
             var predicate = AllEarningsAreForSameProvider(earnings)
-                ? (Func<EarningEventModel, bool>)IncludeEverythingPredicate
-                : (Func<EarningEventModel, bool>)IncludeApprenticeshipPredicate(activeApprenticeship);
+                ? IncludeEverythingPredicate
+                : IncludeApprenticeshipPredicate(activeApprenticeship);
```

Comment:
Nice.  At some point this didn't compile, complaining about an ambiguity in the methods.  A subsequence refactor must have made it OK.

Reply:
TBH that was not me, I think visual studio must have fixed it somehow.